pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -66,6 +235,8 @@ class MCPServer(AbstractToolset[Any], ABC):
     allow_sampling: bool = True
     max_retries: int = 1
     sampling_model: models.Model | None = None
+    allow_elicitation: bool = True
+    elicitation_callback: ElicitationFnT | None = None
```

Comment:
Please pull this new `elicitation_callback` property out into a new PR so we can merge that pretty quickly and close https://github.com/pydantic/pydantic-ai/issues/2330?